packages/ui/src/app/pages/Proposals/PastProposals.tsx

```diff
@@ -20,6 +21,7 @@ import { ProposalsTabs } from './components/ProposalsTabs'
 export const PastProposals = () => {
   const searchSlot = useRef<HTMLDivElement>(null)
   const [filters, setFilters] = useState(ProposalEmptyFilter)
+  const [_, setCookies] = useCookies()
```

Comment:
```suggestion
  const [, setCookies] = useCookies()
```

Reply:
Hello.
I followed your guideline and tried to test more. In the meantime, I figured out the cookie is not good for saving last search. because if i choose proposer in the UI, the size of search is over 4kb, so it couldn't be saved in cookie. 
So can i use localstorage or sessionStorage instead of cookie? 
I hope to confirm this with you.

Reply:
I see no problem to use ls instead.
Related to https://github.com/Joystream/pioneer/issues/3714#issuecomment-1285260544

Reply:
I updated the code by using LocalStorage, please check it